docker/Dockerfile

```diff
@@ -40,24 +40,36 @@ COPY requirements.txt /requirements.txt
 RUN pip3 install -r requirements.txt
 
 COPY requirements-wheels.txt /requirements-wheels.txt
+COPY requirements-wheels-amd64.txt /requirements-wheels-amd64.txt
```

Comment:
We don't need to copy this unless the build is amd64

Reply:
I am not aware of a way to condition this `COPY` by platform in the Dockerfile.  This is only an intermediate stage and the file shouldn't be in the final image.

Reply:
Yeah I was being silly, disregard that.